torchaudio/transforms.py

```diff
@@ -155,6 +155,11 @@ class GriffinLim(torch.nn.Module):
             Values near 1 can lead to faster convergence, but above 1 may not converge. (Default: ``0.99``)
         length (int, optional): Array length of the expected output. (Default: ``None``)
         rand_init (bool, optional): Initializes phase randomly if True and to zero otherwise. (Default: ``True``)
+
+        Example
+                >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
+                >>> transform = torchaudio.transforms.GriffinLim(n_fft=800)
+                >>> transgriffinlim = transform(waveform)
```

Comment:
This looks incorrect. The input of T.GriffinLim should be a spectrogram. Could you change the waveform to a spectrogram?